package_output/DSIronPython3/pkg.json

```diff
@@ -15,5 +15,5 @@
   "site_url": "https://dynamobim.org/",
   "repository_url": "https://github.com/DynamoDS/Dynamo",
   "contains_binaries": true,
-  "node_libraries": ["python3eval"]
+  "node_libraries": [ "DSIronPython3Empty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" ]
```

Comment:
Does an extension need to specify a node library too? I can't recall but do we need to make dummy node libraries for all extensions?

Reply:
SO this is to avoid having Dynamo load iron python in the default assembly load context right ?

Reply:
Sorry dudes, neither of those guesses are correct 😉 

as it says in the comment
`  /// This class/dll exists so that Dynamo knows this package is loaded,
    /// even though it really only contains an extension.
`

it's to fix a bug @avidit found in the DSIronPython package where Dynamo let you uninstall packages with ONLY extensions loaded. Ive filed a bug fix for it.

Reply:
and having python3eval as the node_libs did not solve that issue ? or is the problem that it loads it in the default load context ?

Reply:
`or is the problem that it loads it in the default load context ?` - yep, you got it, we don't want Dynamo to load that assembly itself. We want to load it into the ALC.

Reply:
One more thing. If we do implement ALC on the package side, I wonder if it will work with a future version of DYnamo which will load packages in isolated assem contexts by default 
We will probably need to do only one

Reply:
I was thinking we'd somehow allow packages to opt in to this behavior, definitely more to think about here.